ac/ac-imgClass/src/imgClassifier.jsx

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import React from 'react';
+import Mousetrap from 'mousetrap';
+import type { ActivityRunnerT } from 'frog-utils';
+import styled from 'styled-components';
+
+const ImgBis = styled.img`
+  max-width: 80%;
+  max-height: 100%;
+  position: relative;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const FlexDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 90%;
+`;
+
+const ImgPanel = (props: { url: string }) =>
+  <div style={{ width: '90%', height: '100%' }}>
+    <ImgBis alt="" src={props.url} />{' '}
+  </div>;
+
+const ShortCutPanel = (props: { categories: Array<string> }) =>
+  <ul className="list-group" style={{ width: '20%' }}>
+    {props.categories.map((x, i) =>
+      <li key={x} className="list-group-item">
+        {' '}{i + 1} <span className="glyphicon glyphicon-arrow-right" /> {x}{' '}
+      </li>
+    )}
+  </ul>;
+
+export default ({ activityData, data, dataFn }: ActivityRunnerT) => {
+  const categ = activityData.config.categories || [];
+  const imgs = activityData.config.images || [];
+  categ.forEach((x, i) =>
+    Mousetrap.bind((i + 1).toString(), () => {
+      dataFn.objInsert(x, data.index + 1);
```

Comment:
Also, we might want the list of urls to come from an operator.
In that case you should implement the `mergeFunction` to write in the reactive data both the list from the operator and the list from the config.

In the future we would like to even implement streaming operators which create input data repeatedly. for example half of the students keep generating images and the other half keep categorizing the images. I believe we would manage this through the merge function being called several times.